packages/solo/public/wallet-bridge.html

```diff
@@ -154,11 +154,13 @@
         });
       };
 
-      // Start the flow of messages.
-      if (window.parent !== window) {
-        window.parent.postMessage({ type: 'walletBridgeLoaded' }, '*');
-      }
-      retryWebSocket();
+      window.addEventListener('load', () => {
```

Comment:
This ensures the message wont be posted until after the iframe's "onLoad" event fires. This is the way it works with the react bridge.jsx, so we can rely on the consistent ordering of events in the WalletConnection component.